package.json

```diff
@@ -171,7 +171,7 @@
   ],
   "release": {
     "branches": [
-      "develop"
+      "main"
```

Comment:
I didn't add the `@typeform/semantic-release-config` package since this repo is public and it has no auth to our registry yet. This line should do the work though